ricecooker/classes/nodes.py

```diff
@@ -863,6 +867,7 @@ def validate(self):
         from .files import AudioFile
 
         try:
+            self.duration = audio_files[0].get_duration()
```

Comment:
I don't see where `audio_files` is defined here? Did you forget to add the list comprehension above?

Reply:
When i merge the code it was delete by mistake.